### PR TITLE
satellite/console: add README.md for Project Management API

### PR DESCRIPTION
**Pull request by ihaid**
What: 
closes https://github.com/storj/storj/issues/4741
Why:

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [ ] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/main/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/main/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?
 - [ ] Do the database access patterns make sense?

### CODE REVIEW

File: satellite/console/consoleweb/consoleapi/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+Get users project. 
+
+A successful response body:
+
+```json
+{
+
+}
+```
+### GET /bucket-rollup/projectID={}&bucket={}&since={Unix Timestamp}&before={Unix Timestamp}
```
Comment (Erikvv) [inline satellite/console/consoleweb/consoleapi/README.md:47]: I like RFC 3339 datetimes for the sake of readability

File: satellite/console/consoleweb/consoleapi/README.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+### GET /bucket-rollups?projectID={}&since={Unix Timestamp}&before={Unix Timestamp}	
+Gets project's all buckets usage
+A successful response body:
+
+```json
+{
+
+}
+```
+### PUT /create
```
Comment (Erikvv) [inline satellite/console/consoleweb/consoleapi/README.md:65]: PUT should refer to a resource path with know id  `PUT /project/1234`, else prefer POST.

File: satellite/console/consoleweb/consoleapi/README.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+}
+```
+A successful response body:
+
+```json
+{
+
+}
+```
+### PATCH /update
```
Comment (Erikvv) [inline satellite/console/consoleweb/consoleapi/README.md:80]: Same as PUT, `PATCH /project/1234`, if not possible prefer an RPC-like call with POST

File: satellite/console/consoleweb/consoleapi/README.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+  "detail": ""
+}
+```
+## Project Management API Endpoints
+
+Example of request
+```bash
+curl  -i -L \
+    -H "Accept: application/json" \
+    -H 'Authorization: <key>' \
```
Comment (Erikvv) [inline satellite/console/consoleweb/consoleapi/README.md:32]: For clarity one could include the method: `-H 'Authorization: <method> <token>'`

Like `Authorization: Basic QWxhZGRpbjpvcGVuIHNlc2FtZQ==` or `Authorization: Bearer s07snpnaqiyla0fz1vjg2sam01j21z`

**Review by Erikvv — COMMENTED** [review]
Just some advice, hope you don't mind.

**Comment by ihaid** [comment]
> Just some advice, hope you don't mind.

Actually the API already exists and I wanted to add just some documentation for it

**Comment by littleskunk** [comment]
There are currently some security concerns. The project ID is used as our encryption salt which means the customer should not leak it. Eventually, we are going to use a different encryption salt. That should fix the issue. In the meantime we should add a warning to the documentation so that the customer understands the risk and doesn't leak the project ID.

**Comment by ybutko-ledger** [comment]
Hey @Erikvv! I fixed your comments and added documentation here
https://review.dev.storj.io/c/storj/storj/+/7382

**Comment by VitaliiShpital** [comment]
Hey @Erikvv! I fixed your comments and added documentation here
https://review.dev.storj.io/c/storj/storj/+/7382

**Pull request closed without merging**